java/bulk-export/bulk-export-core/src/main/java/sleeper/bulkexport/core/BulkExportQuerySplitter.java

```diff
@@ -104,11 +104,12 @@ private void init(List<Partition> partitions, Map<String, List<String>> partitio
      * @return                 a list of {@link LeafPartitionQuery}s
      */
     public List<BulkExportLeafPartitionQuery> splitIntoLeafPartitionQueries(BulkExportQuery bulkExportQuery) {
-        LOGGER.debug("There are {} relevant leaf partitions", leafPartitions.size());
+        LOGGER.info("There are {} relevant leaf partitions", leafPartitions.size());
 
         List<BulkExportLeafPartitionQuery> leafPartitionQueriesList = new ArrayList<>();
         for (Partition partition : leafPartitions) {
             List<String> files = getFiles(partition);
+            LOGGER.info("Found {} files for partition {}", files.size(), partition.getId());
```

Comment:
This duplicates the log call just below, "No files for partition {}". We can either move it after that if statement so it's one or the other, or we can remove the other call.

My previous comment mentioned this potential duplication: https://github.com/gchq/sleeper/pull/5888#discussion_r2454244781